### DISCUSSION TITLE
Support for adding data-* attributes

### DISCUSSION THREAD

**Discussion by man-trackunit** [gpbl/react-day-picker#1602] [Ideas]
Adding `data-*` attributes currently fails silently. Would expect that for example a `data-testid="test"` would just get added to the root element.
![CleanShot 2022-11-08 at 14 11 49](https://user-images.githubusercontent.com/79450095/200573573-eee39f0f-7617-4a10-94f2-7321e57429b3.png)

**Comment by gpbl** [comment]
> Adding data-* attributes currently fails silently.

This is surprising ! Will check why...

**Reply by man-trackunit** [reply]
Thanks a lot 🙏

**Comment by gpbl** [comment]
Done in v8.7.0
